tests/python/contrib/test_hexagon/unit_tests.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
All the other changes look good to me, but I can't place where this one comes from.  Is there a different in-progress PR that this should be a part of?

Reply:
Never mind, saw @mehrdad's comment.  Agreed, should remove this file.